contrib/cirrus/setup.sh

```diff
@@ -2,118 +2,47 @@
 
 set -e
 
-# N/B: Most development-related packages are pulled in automatically by
-#      'build-essential' (on ubuntu), for Feodra the groups
-#      '@C Development Tools and Libraries' and '@Development Tools'
-#      are similar.
+# N/B: In most (but not all) cases, these packages will already be installed
+# in the VM image at build-time (from libpod repo.).  Running package install
+# again here, ensures that all cases are covered, and there is never any
+# expectation mismatch.
+source $(dirname $0)/lib.sh
 
-UBUNTU_PACKAGES="
-    aufs-tools
-    bats
-    bzip2
-    coreutils
-    curl
-    git
-    go-md2man
-    golang
-    libdevmapper-dev
-    libglib2.0-dev
-    libgpgme11-dev
-    libseccomp-dev
-    libselinux-dev
-    openssl
-    podman
-    jq
-    netcat
-    rsync
-    runc
-    scons
-    vim
-    wget
-    yum-utils
-    zlib1g-dev
-    xz-utils
-"
+req_env_var OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE
 
-FEDORA_PACKAGES="
-    bats
-    btrfs-progs-devel
-    bzip2
-    containers-common
-    device-mapper-devel
-    findutils
-    git
-    glib2-devel
-    glibc-static
-    gnupg
-    go-md2man
-    golang
-    gpgme-devel
-    libassuan-devel
-    libseccomp-devel
-    make
-    nmap-ncat
-    ostree-devel
-    podman
-    jq
-    rsync
-    runc
-    skopeo-containers
-    wget
-    xz
-"
+echo "Setting up $OS_RELEASE_ID $OS_RELEASE_VER"
+cd $GOSRC
+case "$OS_RELEASE_ID" in
+    fedora)
+        # This can be removed when the kernel bug fix is included in Fedora
+        if [[ $OS_RELEASE_VER -le 32 ]] && [[ -z "$CONTAINER" ]]; then
+            warn "Switching io scheduler to 'deadline' to avoid RHBZ 1767539"
+            warn "aka https://bugzilla.kernel.org/show_bug.cgi?id=205447"
+            echo "mq-deadline" | sudo tee /sys/block/sda/queue/scheduler > /dev/null
+            echo -n "IO Scheduler set to: "
+            $SUDO cat /sys/block/sda/queue/scheduler
+        fi
 
-source $(dirname $0)/lib.sh
+        # Not executing IN_PODMAN container
+        if [[ -z "$CONTAINER" ]]; then
+            warn "Adding secondary testing partition & growing root filesystem"
+            bash $SCRIPT_BASE/add_second_partition.sh
+        fi
 
-install_ooe
+        warn "Buildah testing requires runc"
```

Comment:
After Dan's #2334, I'm not sure this is true any more. I'm going to run some tests to find out, it may be an hour. I'll check back in when I know.

Reply:
oh right, leftover from before debugging this issue.  Thx I'll remove it.